src/provider/store/Actions.ts

```diff
@@ -139,13 +146,21 @@ export const ActionHandlers: ActionHandlerMap = {
     },
     [Action.TOGGLE_VISIBILITY]: (state: RootState, action: ToggleVisibility): RootState => {
         const windowVisible = (action.visible !== undefined) ? action.visible : !state.windowVisible;
-        const storage = Injector.get<'DATABASE'>(Inject.DATABASE);
 
-        storage.get(CollectionMap.SETTINGS).upsert({id: SettingsMap.WINDOW_VISIBLE, value: windowVisible});
-
-        return {
-            ...state,
-            windowVisible
-        };
+        return setCenterVisibility(state, windowVisible);
+    },
+    [Action.BLUR_CENTER]: (state: RootState, action: BlurCenter): RootState => {
+        return setCenterVisibility(state, false);
     }
 };
+
+function setCenterVisibility(state: RootState, windowVisible: boolean): RootState {
+    const storage = Injector.get<'DATABASE'>(Inject.DATABASE);
+
+    storage.get(CollectionMap.SETTINGS).upsert({id: SettingsMap.WINDOW_VISIBLE, value: windowVisible});
```

Comment:
We don't need to store the visiblity in the DB, this is probably just left over code that can be removed.

Reply:
My assumption was that we were going to use this at some point to determine state at startup. Do we definitely not want this?

Reply:
I put it in there to show that it could be done but we took it out. Its definitely something we don't want. I think the storage stuff was just kept so we had a settings DB for when we implement settings down the road.